app/controllers/invitations_controller.rb

```diff
@@ -36,6 +36,17 @@ def resource_params
 
   # Overwrite Devise::Invitations resource_class method
   def resource_class
-    params.dig(:invitation, :type).constantize
+    return params.dig(:invitation, :type).constantize if params.dig(:invitation, :type).present?
+
+    User
   end
+
+  # This method is called to validate the token. It can be overwritten.
+  # def resource_from_invitation_token
+  #   unless params[:invitation_token] &&
+  #     self.resource = resource_class.find_by_invitation_token(params[:invitation_token], true)
+  #     set_flash_message(:alert, :invitation_token_invalid) if is_flashing_format?
+  #     redirect_to after_sign_out_path_for(resource_name)
+  #   end
+  # end
```

Comment:
Yes, I want to leave this here, because we do need to override this method. I'm going to update the comment. Once this is merged in I'm going to get a new issue created so that we can get this fixed.

Reply:
Okay, I ended up hacking something together so that an error message is shown on an invalid token.